src/utils/hdfs_mods.py

```diff
@@ -5,6 +5,9 @@
 import pydoop.hdfs as hdfs
 import pandas as pd
 import json
+import os
+
+from src.data_validation.validation import validationlogger
```

Comment:
This imports a pre-existing logger from the validation module. This means that any error/log message that is outputted points to the wrong module. Try this:

import logging

hdfs_logger = logging.getLogger(__name__)

and rename all validationloggers to the new hdfs_logger